### PR TITLE
Add version.py and ai_edge_torch.__version__

### PR DESCRIPTION
**Pull request by chunnienc**
Allow `print(ai_edge_torch.__version__)`

BUG=https://buganizer.corp.google.com/issues/357076369

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,11 +34,19 @@
...
+# TODO(b/357076369): move version updating logics to version.py
+version_py = here / "ai_edge_torch" / "version.py"
+version_regex = "__version__\s*=\s*(\"|')(?P<version>[^\"']+)(\"|')"
+version = re.search(version_regex, version_py.read_text()).group("version")
+
 if nightly_release_date := os.environ.get("NIGHTLY_RELEASE_DATE"):
   name += "-nightly"
   version += ".dev" + nightly_release_date
-
+  version_py.write_text(
```
Comment (majiddadashi) [inline setup.py:45]: is the code here just to canonicalize the contents of __version__.py in terms of quotes? why is this exactly needed?

Comment (chunnienc) [reply setup.py:45]: It updates the content of `version.py` to `"0.2.0.devxxxxxxxx"`. The modified file will be packed to the wheel and user would get that updated value from `ai_edge_torch.__version__`

**Comment by chunnienc** [comment]
@majiddadashi @talumbau does this look good? I'd like to cherry pick this to today's release.

**Review by majiddadashi — APPROVED** [review]

**Comment by chunnienc** [comment]
@talumbau I merged this PR for the release. Feel free to comment on this. I can send follow up PRs to improve this if needed.

**Pull request merged**